op-batcher/batcher/driver.go

```diff
@@ -668,7 +681,7 @@ func (l *BatchSubmitter) clearState(ctx context.Context) {
 			}
 		case <-ctx.Done():
 			l.Log.Warn("Clearing state cancelled")
-			l.state.Clear(eth.BlockID{})
+			l.channelMgr.Clear(eth.BlockID{})
```

Comment:
don't we need to guard this call with the mutex?

Reply:
Same as above, this function is only called:

* by StartBatchSubmitting before any goroutines are launched and
* by syncAndPublish, which is already holding the lock

So we shouldn't add a guard here, because it is unecessary and would cause a deadlock.